app/src/main/java/com/zeapo/pwdstore/utils/Extensions.kt

```diff
@@ -135,9 +131,14 @@ suspend fun FragmentActivity.commitChange(
     }.execute()
 }
 
-fun FragmentActivity.isPermissionGranted(permission: String): Boolean {
-    return ContextCompat.checkSelfPermission(this, permission) == PackageManager.PERMISSION_GRANTED
-}
+fun FragmentActivity.isPermissionGranted(permission: String): Boolean =
+    ContextCompat.checkSelfPermission(this, permission) == PackageManager.PERMISSION_GRANTED
+
+
+fun Fragment.isPermissionGranted(permission: String): Boolean =
+    ContextCompat.checkSelfPermission(requireActivity(), permission) == PackageManager.PERMISSION_GRANTED
+
+fun Fragment.finish() = requireActivity().finish()
```

Comment:
Move to `FragmentExtensions` along with the extensions at the bottom of this file.